app/services/link_checker_api_service.rb

```diff
@@ -11,11 +11,23 @@ def self.check_links(reportable, webhook_uri)
     uris = extract_links(reportable)
     raise "Reportable has no links to check" unless uris.count > 0
 
+    Rails.logger.info "Calling create_batch for Edition #{reportable.id}"
+
     batch_report = Whitehall.link_checker_api_client.create_batch(
       uris,
       webhook_uri: webhook_uri,
       webhook_secret_token: webhook_secret_token
     )
+
+    Rails.logger.info "Batch for Edition #{reportable.id} with id #{batch_report.id}"
+
+    existing_report = LinkCheckerApiReport.find_by(batch_id: batch_report.id)
+
+    if existing_report
+      Rails.logger.info "LinkCheckerApiReport exists for #{batch_report.id}"
+      Rails.logger.info "It belongs to Edition #{existing_report.link_reportable.id} not #{reportable.id}"
```

Comment:
Can we just use `existing_report.link_reportable` and `reportable` in the string then we might get info such as the type of document.